tests/unit_tests/test_wave_01.py

```diff
@@ -40,16 +40,17 @@ def test_removing_from_inventory_returns_item():
     assert item not in vendor.inventory
     assert result == item
 
-@pytest.mark.skip
+# @pytest.mark.skip
 def test_removing_not_found_is_false():
     item = "item to remove"
     vendor = Vendor(
         inventory=["a", "b", "c"]
     )
 
-    result = vendor.remove(item)
+    result = False
```

Comment:
Your test has no act section. Line 50 was changed to hardcode the value of `result` to `False` instead of invoking the `remove` method. You aren't actually testing the `remove` method here.